### DISCUSSION TITLE
SecretTribeDev?

### DISCUSSION THREAD

**Discussion by ObiKeahloa** [monkeytypegame/monkeytype#1760] [Questions] [answered]
Is this some kind of a hidden website? @Miodec 
Have I found something I shouldn't? It has an extra menu named "tribe"
![image](https://user-images.githubusercontent.com/63236194/130628393-b9ce3c1c-3524-4df7-9917-956327fa1142.png)

**Comment by Miodec** [comment, marked as answer]
Yeah that's fine, it's been publicly available for a while now.

**Comment by frogfrogfrogfrogfrog** [comment]
Oh yeah I remember finding Monkey Dev when Monkeytype still couldn't type accented characters or something
